image-layout.md

```diff
@@ -115,4 +116,42 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## Index 
+
+Index is intended to be generated from the content in an image-layout after any updates have been made.
+Allows for mapping random access to the references in the image-layout, and needing only to read from the `oci-layout` once to build this map of references.
+
+**index** *array of objects*, OPTIONAL
+
+  Provides a single access to top-level information about the layout.
+  Each object has the following properties:
+
+  - **ref** *string*, REQUIRED
+  
+    The name of the ref (correspond JSON descriptor found in `./refs/`)
+
+  - **annotations** *string-string map*, OPTIONAL
+
+    This field is under the same constaints as [manifest `annotations`](manifest.md#image-manifest-property-descriptions)
```

Comment:
I'd rather not add this field to the ref index, since it wouldn't be supported by `ref/`-walking listers.  Can you motivate a use-case for it?  Folks who wanted to annotate a ref can already do so by adding a new ref (e.g. `v1.0.0.annotations`) pointing at a CAS blob with their desired payload.